create-publication-from-doi/src/main/java/no/unit/nva/doi/fetch/MainHandler.java

```diff
@@ -100,11 +101,7 @@ public MainHandler(PublicationConverter publicationConverter,
 
     @JacocoGenerated
     private static MetadataService getMetadataService() {
-        try {
-            return new MetadataService();
-        } catch (IOException e) {
-            throw new RuntimeException("Error creating handler", e);
-        }
+        return attempt((Callable<MetadataService>) MetadataService::new).orElseThrow();
```

Comment:
For the sake of readability I would prefer the try/catch style.
`.orElseThrow()` throw what?

Reply:
It follows the pattern of of Optional.orElseThrow(). It catches the Exception and wraps it in a RuntimeException if it is a checked Exception or throws it as it is if it a RuntimeException.

Reply:
Yes I know. But that does not make it more readable.
It is much more fluid (like spoken language) if you have something saying **try** to make **new MetadataService** and **catch** if **exception** occurs and **throw a RuntimeException**.
Instead of saying:
**attempt** to **call** MetadataService **MetadataService** **new** or if  **throw** ...